client/my-sites/checkout/composite-checkout/components/wp-checkout-order-review.tsx

```diff
@@ -70,8 +66,6 @@ const CouponEnableButton = styled.button`
 export default function WPCheckoutOrderReview( {
 	className,
 	removeProductFromCart,
-	removeCoupon,
-	couponStatus,
```

Comment:
These can be imported directly from `useShoppingCart` so we don't need to inject them here. (Whereas `removeProductFromCart` is doing some extra work so we need that.)